gui/bar.py

```diff
@@ -0,0 +1,15 @@
+import pygame
+
+
+class Bar():
+    def __init__(self, x, y, points, max_points):
+        self.x = x
+        self.y = y
+        self.points = points
+        self.max_points = max_points
+
+    def draw(self, points, screen, colour_of_background_bar, colour_of_front_bar):
+        self.points = points
+        ratio = self.points / self.max_points
+        pygame.draw.rect(screen, colour_of_background_bar, (self.x, self.y, 150, 20))
+        pygame.draw.rect(screen, colour_of_front_bar, (self.x, self.y, 150 * ratio, 20))
```

Comment:
There is no need to provide screen as an argument, `pygame.display.get_surface()` will automatically get current screen. Also colours probably won't be changed each time the bar will be drawn. You can set colours once - during object initialization, so `draw` method will be checking only actual points change.
```suggestion
class Bar():
    def __init__(self, x, y, points, max_points, colour_of_background_bar, colour_of_front_bar):
        self.x = x
        self.y = y
        self.points = points
        self.max_points = max_points
        self.colour_of_background_bar = colour_of_background_bar
        self.colour_of_front_bar = colour_of_front_bar
        self.screen = pygame.display.get_surface()

    def draw(self, points):
        self.points = points
        ratio = self.points / self.max_points
        pygame.draw.rect(self.screen, colour_of_background_bar, (self.x, self.y, 150, 20))
        pygame.draw.rect(self.screen, colour_of_front_bar, (self.x, self.y, 150 * ratio, 20))
```